modules/sdk-unified-wallet/package.json

```diff
@@ -14,10 +14,11 @@
   },
   "dependencies": {
     "@bitgo/sdk-core": "^5.2.0",
-    "@bitgo/statics": "^14.1.0"
+    "@bitgo/statics": "^14.1.0",
+    "bitgo": "^16.4.0",
+    "openpgp": "5.1.0"
```

Comment:
this belongs in devDependencies since it's only used by your unit tests, please move there.